experimental/sample_web_static/index.html

```diff
@@ -0,0 +1,225 @@
+<!DOCTYPE html>
+<html>
+
+<!--
+Copyright 2022 The IREE Authors
+
+Licensed under the Apache License v2.0 with LLVM Exceptions.
+See https://llvm.org/LICENSE.txt for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<head>
+  <meta charset="utf-8" />
+  <title>IREE Static Web Sample</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- TODO(scotttodd): use local copy for CORS webserver / SharedArrayBuffer workarounds? -->
+  <script src="https://code.createjs.com/1.0.0/easeljs.min.js"></script>
```

Comment:
in future revs when we want multithreading to work you could wget this into a `.tmp/` in the serving path (so we can .gitignore it) as part of the build script if it doesn't exist

Reply:
Yep! That's what I was thinking too.

I also miss the js_dev_server stuff that was built for Bazel a bit... that made it pretty easy to bundle and server deps like this. I'm sure there's some webpack/nodejs/npm/whatever thing in open source land but at the scale of this sample a few tweaks to the script should be good enough :P